frontend/src/metabase-types/api/permissions.ts

```diff
@@ -43,10 +52,11 @@ export type DownloadTablePermission =
   | { [key: TableId]: DownloadPermission };
 
 export type DatabasePermissions = {
-  data: DatabaseAccessPermissions;
-  "data-model"?: DataModelPermissions;
-  download?: DownloadAccessPermission;
-  details?: DetailsPermissions;
+  [DataPermission.VIEW_DATA]: SchemasPermissions;
+  [DataPermission.CREATE_QUERIES]?: NativePermissions;
+  [DataPermission.DATA_MODEL]?: DataModelPermissions;
+  [DataPermission.DOWNLOAD]?: DownloadAccessPermission;
+  [DataPermission.DETAILS]?: DetailsPermissions;
```

Comment:
Since DataPermission is a type, can it be used as a value here?

Reply:
`DataPermission` is an enum so yes

Reply:
Peculiar. Earlier today my linter (coc.nvim) told me that DataPermission.VIEW_DATA, DataPermission.CREATE_QUERIES, etc., couldn't be used as values in this file because DataPermissions is imported as a type. But now it's not complaining anymore. I wonder if my linter is not configured properly.